src/garage/torch/algos/vpg.py

```diff
@@ -148,41 +157,42 @@ def train_once(self, itr, paths):
         obs_flat = torch.cat(filter_valids(obs, valids))
         actions_flat = torch.cat(filter_valids(actions, valids))
         rewards_flat = torch.cat(filter_valids(rewards, valids))
-        advantages_flat = self._compute_advantage(rewards, valids, baselines)
+        returns_flat = torch.cat(filter_valids(returns, valids))
```

Comment:
You have an extra line here.

Reply:
You still have an extra line here, as far as I can tell.

Reply:
I'm sorry I don't know what you are referring to. Could you clarify what the extra line means?

Reply:
I'm sorry I don't know what you are referring to. Could you clarify what the extra line means?

Reply:
Okay, looks like the diff was just glitched on my end. Sorry for the confusion.